app/controllers/user_controller.php

```diff
@@ -81,9 +82,49 @@ public function isMatchPassword()
 
     public function profile()
     {
-        if (!$_SESSION['username']) {
+        if (!Session::get('username')) {
             redirect(url('user/login'));
         }
+
+        $user = User::getAll();
+        $comments = User::getComments();
+        $users = new User;
+        $image = User::getImage(); 
+        $error = false;
+
+        if (isset($_FILES["image"])) {
+            $target_dir = "image/";
+            $target_file = $image;
+            $image_file_type = pathinfo($target_file,PATHINFO_EXTENSION);
+            $target_file = $target_dir . $_SESSION['username'].".{$image_file_type}";
+            $is_uploaded = 1;
+
+            if (isset($_POST["submit"])) {
+                if (getimagesize($_FILES["image"]["tmp_name"]) !== false) {
+                    $is_uploaded = 1;
+                } else {
```

Comment:
please avoid nested conditional statements :bow: 
I think it would be better to separate this part into another function instead :dart: That way it will also be reusable.